fastlane/metadata/steam/indonesian/description.txt

```diff
@@ -1,62 +1,62 @@
-Mindustry is a factory-building game with tower defense and RTS elements. Create elaborate supply chains to feed ammo into your turrets, produce materials to use for building, and construct units. Command units to capture enemy bases, and expand your production. Defend your core from waves of enemies.
+Mindustry adalah sebuah game manajemen pembangunan industri dengan elemen RTS dan menara pertahanan. Buat rantai pasokan yang rumit untuk memasukkan amunisi ke menara Anda, menghasilkan material yang digunakan untuk membangun, dan membuat unit. Perintahkan unit untuk merebut markas musuh, dan perluas produksi Anda. Pertahankan inti Anda dari gelombang musuh.
 
-Play through the campaign alone or with with your friends in cross-platform multiplayer co-op and challenge them in team-based PvP matches.
+Mainkan kampanye sendirian atau bersama teman-teman Anda dalam lintas platfrom multi pemain co-op dan tantang mereka dalam pertandingan PvP berbasis tim.
 
 [img]{STEAM_APP_IMAGE}/extras/factory.gif[/img]
 
-[h2]Factory Building[/h2]
+[h2]Bangunan Pabrik[/h2]
 
 [list]
-[*] Create elaborate supply chains of conveyor belts to feed ammo into your turrets and produce materials to use for building
-[*] Use production blocks to create a wide variety of advanced materials
-[*] Distribute liquids and fight constant challenges, like outbreaks of fire or enemy flier raids
-[*] Get the most out of your production by supplying optional coolant and lubricant
-[*] Produce a wide variety of units for automatic management of your base or assault on enemy bases
+[*] Buat rantai pasokan yang rumit untuk memasukkan amunisi ke menara Anda dan menghasilkan bahan yang digunakan untuk membangun
+[*] Gunakan blok produksi untuk membuat berbagai macam material canggih
+[*] Mendistribusikan cairan dan melawan tantangan konstan, seperti pabrik terbakar atau serangan udara musuh

```

Comment:
"outbreak of fire" can be translit to "bencana kebakaran"
"pabrik terbakar" is too generic, because Steam Gen also could burn if in contact with flames